### PR TITLE
Fixed up Quad Cannon draw code

### PR DESCRIPTION
**Pull request by Generalcamo**
- Fixed radar dish animation wasn't showing. ~commy2
- Level 1 salvaged quad cannon now properly uses damaged model
- Corrected dust trail to use the Quad Cannon's particle rather than the Rocket buggy's

### CODE REVIEW

File: Patch104pZH/GameFilesEdited/Data/INI/Object/ChemicalGeneral.ini [hunk 1]
```diff
@@ -16607,11 +16607,14 @@ Object Chem_GLAVehicleQuadCannon
   ;UpgradeCameo4 = XXX
   ;UpgradeCameo5 = XXX
 
+  ; Patch 1.04p @bugfix GeneralCamo 13/09/2021 - Fixed animations, set dust trail to the correct particle, and corrected salvage level 1 to properly use damaged model
```
Comment (xezon) [inline Patch104pZH/GameFilesEdited/Data/INI/Object/ChemicalGeneral.ini:16610]: Please use `Patch104p` as opposed to `Patch 1.04p` in your comments so it is consistent with the keyword of all other comments we added.

Comment (xezon) [reply Patch104pZH/GameFilesEdited/Data/INI/Object/ChemicalGeneral.ini:16610]: Other than that it looks all good.

**Review by xezon — CHANGES_REQUESTED** [review]

**Comment by commy2** [comment]
What does "animation playback" mean here?

This is missing the GC Quads.

**Comment by Generalcamo** [comment]
@xezon Will commit a fix.

@commy2 I'll add the GC Quads and the CINE quads. Specifically, the Quad Cannons have an animation for their radar dishes. It currently does not play at all however, and this code fixes that.

**Comment by commy2** [comment]
Nice, dishes. Yumm.

No need to fix the CINE ones.

**Pull request closed without merging**